dist/index.html

```diff
@@ -13,25 +13,17 @@ <h1 class="header">LeaderBoard</h1>
         <div class="cont-content">
             <div class="score-sect">
                 <h2>Recent scores</h2>
-                <button>Refresh</button>
+                <button id="btn-get-data">Refresh</button>
             </div>
             <div class="board">
-                <ul>
-                    <li>Name: 100</li>
-                    <li>Name: 20</li>
-                    <li>Name: 50</li>
-                    <li>Name: 78</li>
-                    <li>Name: 125</li>
-                    <li>Name: 42</li>
-                </ul>
             </div>
         </div>
      </div>
      <div class="second-container">
         <h3>Add your score</h3>
-        <form action="">
-            <input type="text" placeholder="Your name">
-            <input type="number" placeholder="Your score">
+        <form id="form" action="">
+            <input id="name" type="text" placeholder="Your name">
+            <input id="score" type="number" placeholder="Your score">
             <button>Submit</button>
```

Comment:
- [ ] [OPTIONAL] Kindly clear the inputs field after pressing the submit button for a better user experience.